src/css/themes/flatly/infragistics.theme.css

```diff
@@ -2550,7 +2552,7 @@ body .ui-tooltip {
 .ui-iggrid .ui-iggrid-mergedcellsbottom.ui-state-highlight {
   border-bottom-color: #ecf0f1;
 }
```

Comment:
Again the change was committed in .less and now I commit the generated CSS file.